_data/ispn.yml

```diff
@@ -193,20 +193,20 @@ old:
       configdocs: https://docs.jboss.org/infinispan/10.0/configdocs/
       jmxdocs: https://docs.jboss.org/infinispan/10.0/apidocs/jmxComponents.html
 
-  9.4.20:
+  9.4.23:
     minor_version: 9.4
-    version: 9.4.20.Final
-    maven_latest: 9.4.20.Final
-    release_date: 2020/08/05
+    version: 9.4.23.Final
+    maven_latest: 9.4.23.Final
+    release_date: 2021/05/14
     fixed_issues: https://issues.redhat.com/secure/ReleaseNote.jspa?projectId=12310799&version=12345567
     codename: Infinity Minus ONE +2
     whats_new: Segmented memory/stores, Hot Rod TX recovery, Transcoding everywhere, etc...
     download:
-      all: https://downloads.jboss.org/infinispan/9.4.20.Final/infinispan-9.4.20.Final-all.zip
-      bin: https://downloads.jboss.org/infinispan/9.4.20.Final/infinispan-9.4.20.Final-minimal.zip
-      server: https://downloads.jboss.org/infinispan/9.4.20.Final/infinispan-server-9.4.20.Final.zip
-      wfmodules: https://downloads.jboss.org/infinispan/9.4.20.Final/infinispan-wildfly-modules-9.4.20.Final.zip
-      docker: jboss/infinispan-server:9.4.20.Final
+      all: https://downloads.jboss.org/infinispan/9.4.23.Final/infinispan-9.4.23.Final-all.zip
+      bin: https://downloads.jboss.org/infinispan/9.4.23.Final/infinispan-9.4.23.Final-minimal.zip
+      server: https://downloads.jboss.org/infinispan/9.4.23.Final/infinispan-server-9.4.23.Final.zip
+      wfmodules: https://downloads.jboss.org/infinispan/9.4.23.Final/infinispan-wildfly-modules-9.4.23.Final.zip
+      docker: jboss/infinispan-server:9.4.23.Final
```

Comment:
@ryanemerson are we going to have a new image for 9.4 or should I revert this change?

Reply:
New images only for `12.1.x` and `13.x`